airbyte-integrations/connectors/destination-mysql/src/main/java/io/airbyte/integrations/destination/mysql/MySQLDestination.java

```diff
@@ -73,7 +73,7 @@ public AirbyteConnectionStatus check(final JsonNode config) {
       }
 
       return new AirbyteConnectionStatus().withStatus(Status.SUCCEEDED);
-    } catch (final ConnectionErrorException e) {
+    } catch (final ConfigErrorException e) {
       final String message = getErrorMessage(e.getStateCode(), e.getErrorCode(), e.getExceptionMessage(), e);
```

Comment:
Same thing here. This would fail right since `ConfigErrorException` does not have the `getStateCode()` and various methods so it would need to call the `e.getDisplayMessage()` instead